lagom-client-restaurant/menu-item-impl/src/main/resources/application.prod.conf

```diff
@@ -2,63 +2,18 @@ include "application.conf"
 
 play.http.secret.key = "unused secret"
 
-http {
-  address = ${?HTTP_BIND_ADDRESS}
-  port = 9000
-}
```

Comment:
This is the default port, and was set to the default address (0.0.0.0) so there's no need to override.